e2e/e2etests/test_bitcoin_withdraw.go

```diff
@@ -32,6 +32,16 @@ func WithdrawBitcoin(r *runner.E2ERunner) {
 	// mine blocks
 	stop := r.MineBlocks()
 
+	// withdraw 0.1 BTC from ZRC20 to BTC legacy address
+	tx, err = r.BTCZRC20.Withdraw(r.ZevmAuth, []byte("1EYVvXLusCxtVuEwoYvWRyN5EZTXwPVvo3"), amount)
+	if err != nil {
+		panic(err)
+	}
+	receipt = utils.MustWaitForTxReceipt(r.Ctx, r.ZevmClient, tx, r.Logger, r.ReceiptTimeout)
+	if receipt.Status == 1 {
+		panic(fmt.Errorf("withdraw receipt status is successfull for an invalid BTC address"))
+	}
```

Comment:
We should move this one to a new test like `WithdrawBitcoinInvalidAddress`

Since we we can now run tests on live networks we wouldn't want to take the risk of losing this value if we don't want to test this one.

Reply:
I don't mind moving it to a different test, but not sure what you mean by losing this value.
Event on a live network , if in case the tx goes through ,it should panic at line 41 , which would indicate there is a problem with the network. 

Do you want to run this test only on test networks ?

Reply:
Bitcoin Withdraw is a test that will be usually run on mainnet when deploying new things or doing major upgrade (we actually used it for mainnet launch) so better to keep it atomic. Just testing a regular withdrawal.

> which would indicate there is a problem with the network.

Then the 0.1 btc could be lost, which is big on mainnet

Reply:
270eadd84e33a355795df538ecfaaa71299c13e0